package.json

```diff
@@ -13,7 +13,8 @@
     "check": "lerna run check --stream",
     "check-evergreen": "lerna run check",
     "bootstrap": "lerna exec --stream -- npm ci --quiet",
-    "bootstrap-evergreen": "lerna exec -- npm ci --quiet",
+    "bootstrap-evergreen": "cross-env ELECTRON_SKIP_BINARY_DOWNLOAD=1 lerna exec -- npm ci --quiet && npm run install-electron",
+    "install-electron": "cross-env ELECTRON_SKIP_BINARY_DOWNLOAD= lerna exec --concurrency 1 -- node ../../scripts/install-electron.js",
```

Comment:
Where is this `ELECTRON_SKIP_BINARY_DOWNLOAD` variable used? Maybe I'm missing it

Reply:
It's not used in our code, but rather by electron itself: https://www.electronjs.org/docs/tutorial/installation#skip-binary-download